src/main/resources/default/templates/wondergem/page-messages.html.pasta

```diff
@@ -1,7 +1,7 @@
 <div class="row">
     <div class="col-md-12" id="messageBox">
         <i:for type="sirius.web.controller.Message" var="msg" items="user.getMessages()">
-            <div class="alert @msg.getType()">
+            <div class="alert @msg.getType().getStyle()">
```

Comment:
Wouldn't it be better to also overwrite toString of the Type enum to return getStyle? 
This make this change to not break as much as '@msg.getType()' would also be valid in templates